### ISSUE TITLE
Add ip option for using in template

### ISSUE THREAD

**Issue by rodehoed** [tuxis-ie/nsedit#58]
Hi,

What we really are missing is an option to add an ip at zone creation. And let us use this IP adress in a template. So we can create records such as www with the specified ip.

- Rodehoed

**Comment by tuxis-ie** [comment]
You mean that ALL records point to that specific IP?

**Comment by rodehoed** [comment]
Yes but better would be the use of eg. a ${ip_cookie} we can use in the template array.

**Comment by sonics6** [comment]
Is it possible to have at the template some placeholders like [ZONE]?

**Comment by tuxis-ie** [comment]
PR #153 does that. Not for IP's though.
